aut-0.50.0/dataframe-schemas.md

```diff
@@ -0,0 +1,157 @@
+# Archives Unleashed Toolkit DataFrames
+
```

Comment:
Yeah we should add a tiny bit of descriptive text here. I assume this is for advanced users? 

Something like:

Below you can find all of the DataFrame schemas found in each object. For example, if you extract `.all` from WARCs, you will see the fields below. Some of the most popular ones include `all` (which includes content, URLs, and file types); `webpages` (which includes full-text content and language); and `webgraph` which includes hyperlink information.